newIDE/app/src/BrowserApp.js

```diff
@@ -70,6 +70,8 @@ export const create = (authentication: Authentication) => {
                 <BrowserS3PreviewLauncher {...props} ref={ref} />
               )}
               initialDialog={appArguments['initial-dialog']}
+              initialGameId={appArguments['game-id']}
+              initialGameTab={appArguments['game-tab']}
```

Comment:
I feel like we should be more accurate: `game-dashboard-tab`?

Reply:
Agree